korge/src/commonMain/kotlin/com/soywiz/korge/view/View.kt

```diff
@@ -978,6 +979,28 @@ class ViewTransform(var view: View) {
 }
 */
 
+/** When included in the tree, this returns the Views singleton. When not attached yet, this will return null. */
+val View.views: Views? get() = stage?.views
+
+/**
+ * Register a callback that will be executed when the specified [this] [View]
+ * is attached to the [Stage].
+ */
+fun View.onAttachToStage(block: Views.(view: View) -> Unit) {
+    // @TODO: On KorGE 2.0 we should have components for subscribing to attaching and detaching from the stage.
+    // @TODO: We would keep a list of views with components attached to the Stage to avoid iterating the tree each time
+    addComponent(object : UpdateComponent {
+        override val view: View = this@onAttachToStage
+        override fun update(ms: Double) {
+            val views = view.views
+            if (views != null) {
+                removeFromView()
+                block(views, view)
+            }
```

Comment:
What if this view is being removed from container and added to it later? This component will work only for the first attachment to the stage.